framework/vendure/fetcher.ts

```diff
@@ -9,12 +9,18 @@ async function getText(res: Response) {
   }
 }
 
+async function fetcherError(options: {
+  status: number
+} & ErrorProps) {
+  return new FetcherError(options)
+}
+
```

Comment:
`FetcherError` already has the `status` prop.

Reply:
Hi,
Not sure it's what you meant but I just removed unnessary new function